apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf

```diff
@@ -47,6 +47,14 @@ auth.pgsql.ssl = off
 ##
 ## auth.pgsql.ssl_opts.tls_versions = tlsv1.2
 
+## TLS version
+## You can configure multi-version use "," split,
+## default value is :tlsv1.2
+## Example:
+##    tlsv1.1,tlsv1.2,tlsv1.3
+##
+## auth.pgsql.ssl.tls_versions = tlsv1.2
```

Comment:
this needs quote.

Reply:
And a lot other places in this config.
Ping @z8674558 to have a look, If you think it's better to fix them in another PR, please help to merge this one.

Reply:
you do not have to quote string with `.` .
```
5> hocon:binary("a=aa.bb").
{ok,#{a => <<"aa.bb">>}}
```